src/app/core/services/dialog.service.ts

```diff
@@ -27,4 +29,27 @@ export class DialogService {
       }
     });
   }
+
+  checkIfFieldIsModified(
+    form: FormGroup,
+    initialField: string,
+    formField: string,
+    issue: Issue,
+    actionIfModified: () => void,
+    actionIfNotModified: () => void
+  ) {
+    const issueTitleInitialValue = issue[initialField] || '';
+    const isModified = form.get(formField).value !== issueTitleInitialValue;
+    this.performActionIfModified(isModified, actionIfModified, actionIfNotModified);
```

Comment:
Since the name of this function is "Check if field is modified", I think we can leave this line out and return ```isModified```. We can call the two functions separately since what we are doing here is passing the last 2 arguments to ```performActionIfModified```

Reply:
Hi, sflr but I made some changes according to your comment. Could you help me see if this is what you're looking for? Thanks in advance!

Reply:
Apologies for the late reply, this looks better 👍